ios/FluentUI/Core/Colors.swift

```diff
@@ -492,7 +496,7 @@ public final class Colors: NSObject {
         return colorProvidersMap.object(forKey: window)?.primaryShade30Color(for: window) ?? FallbackThemeColor.primaryShade30
     }
 
-    private static var colorProvidersMap = NSMapTable<UIWindow, ColorProviding>(keyOptions: .weakMemory, valueOptions: .weakMemory)
+    private static var colorProvidersMap = NSMapTable<UIWindow, ColorProviding>()
```

Comment:
no we shouldn't change the weak memory allocation here. The demo app should be responsible of holding on to provider instances.